src/themes/teams/components/Icon/svg/ProcessedIcons/create-index.sh

```diff
@@ -0,0 +1,30 @@
+export importTemplate="import ICONNAME from './ICONPATH'"
+export exportTemplate="ICONNAME,"
+export template="import { TeamsSvgIconSpec } from '../types'\n\n// IMPORTS\n\nexport default {\n  // EXPORTS\n} as { [iconName: string]: TeamsSvgIconSpec }"
+
+FileName="index-new.ts"
+echo -e "${template}" > $FileName;
+
+for icons in $(ls *.tsx); do
+  echo "Icon is: $icons"
+
+  # GET ICON PATH & NAME
+  Path=${icons/\.tsx/}
+  echo "Path is: $Path"
+
+  UnprocessedName=${Path//icons-/processedIcons_}
+  echo "UnprocessedName is: $UnprocessedName"
+
+  Name=${UnprocessedName//-/}
+  echo "Name is: $Name"
+
+  # REPLACE INSTANCES IN IMPORT & EXPORT TEMPLATES
+  ImportTest=$(echo "$importTemplate" | sed "s/ICONNAME/$Name/" | sed "s/ICONPATH/$Path/")
+  ExportTest=$(echo "$exportTemplate" | sed "s/ICONNAME/$Name/")
+
+  # CREATE IMPORT
+  sed -i "/IMPORT/a  $ImportTest" $FileName
```

Comment:
note that with the following `prepend` processing semantics the order of icons will be _reversed_ alphabetical. So, would suggest either
- ensure that icons are sorted properly at the point when their name is mapped to react elements (would prefer this option)
- or use `-r` flag for `ls` command on this line: https://github.com/stardust-ui/react/pull/478/files#diff-2c8401364afce39e9a7dc9c807267b5aR8